app/queries/low_inventory_query.rb

```diff
@@ -0,0 +1,14 @@
+class LowInventoryQuery
+  attr_reader :organization
+
+  def initialize(organization:)
+    @organization = organization
+  end
+
+  def call
```

Comment:
I prefer the functional method of `def self.call(organization)` rather than using `.new`. It's easier to stub out and needs less indirection.